content/en/contribute/medic/onboarding/all-the-things.md

```diff
@@ -0,0 +1,135 @@
+* Retros - MetroRetro
+* 2 Daily Standups
+* Weekly FWG meetings
+
+### Process
+
+#### Development
+* Basic current process:
+1. Take ticket (GitHub issue from a board)
+1. Code something up on a branch
+1. Open pull request
+1. Move ticket to “Ready for AT (Acceptance Testing)”
+1. Upon testing passing, merge and delete branch
+1. Get QA engineers involved early in the process!
+* Releases
+1. Backwards compatibility matters a lot
+1. Can feel slow at times, but we’re making a lot of progress here. See above about how Focused Groups work.
+1. Quality matters a lot
+* Data engineering -> software engineering
+  * Workflow management 
+  * Dashboards
+* The main repos to look at:
+  * cht-core
+  * cht-conf
+  * cht-android 
+  * cht-sync 
+  * couch2pg
+  * cht-pipeline
+  * cht-sync
+  * cht-monitoring
+* Continuous Integration (CI) with GitHub Actions
+* No CD, as no SaaS setup
+
+#### GitHub 
+* Tons of stuff happens here.
+* Recommendation: [Set up your reminders/notifications](https://medic.slack.com/archives/C024KTGRW/p1617308776092600) 
+* A few important boards: 
+  * [SRE Engineering](https://github.com/orgs/medic/projects/38)
+  * [SRE Support](https://github.com/orgs/medic/projects/19)
+  * [Ecosystem Workstream](https://github.com/orgs/medic/projects/134/views/11)
+  * [Allies Workstream](https://github.com/orgs/medic/projects/134/views/3)
+  * [Care Teams Workstream](https://github.com/orgs/medic/projects/134/views/2)
+  * [Test Automation](https://github.com/orgs/medic/projects/134/views/12)
+
+#### QA
+* Doing a lot of automation
+* Trying to move from manual AT and release testing to fully automated
+* Trying out [quality assistance]({{% ref "quality-assistance" %}})
+  * Faster start-to-live 
+  * Avoiding silos and shifting of responsibilities (coding and quality)
+
+#### SRE
+* Support
+* Ticketing system: only GitHub
+* Not on-call
+* We’re offline first, so not every outage calls for immediate action/resolution
+* Engineering work - Archv3 (cht-core 4.0 version)
+
+### Product
+#### [CHT Forum](https://forum.communityhealthtoolkit.org/)
+* We’re trying to keep this active
+* Encourage teammates to post and answer questions there instead of slack when the public might benefit
+* Expecting you to be proactive and support the team with checking forum posts and helping when questions arise
+
+#### Partners
+* Medic-hosted
+* Self-hosted
+* Technical partners
+
+#### Product Development Process
+* At Medic, we follow [Continuous Discovery]({{% ref "continuous-discovery-overview" %}}) as a [Product Development Process]({{% ref "product-development-process" %}})
+  * Trying to follow closely with the [Continuous Discovery Habits](https://www.producttalk.org/2021/05/continuous-discovery-habits/) book by Teresa Torres
+  * [Focused Working Groups]({{% ref "focused-groups" %}}) within the product team to focus on specific groups of users
+    * Allies
+    * Care Teams
+    * Infrastructure
+    * Ecosystem
```

Comment:
@andrablaj In the other PR you reviewed, I made a separate section for the Product Team. Should we keep it together here, or is it a separate file okay?  

We can polish this PR and merge it to start collaborating; it's okay if it's a list of topics for now; it's already a good start. :)

Reply:
I think the way you organized the page in the other PR is perfect. We can always merge the pages if we change our minds later.